DAO.sol

```diff
@@ -84,7 +84,7 @@ contract DAOInterface {
     uint public proposalDeposit;
 
     // the accumulated sum of all current proposal deposits
-    uint sumOfProposalDeposits;
+    uint public sumOfProposalDeposits;
```

Comment:
Was this done on purpose? Or did you make it public only for testing purposes and forgot it there?